Makefile

```diff
@@ -355,3 +357,53 @@ update_crt_crd: download_crt_crd $(crds)
 check-repo:
 	./hack/check-repo.sh "$(REPO)"
 .PHONY: check-repo
+
+token_version ?= $(shell yq -r '.nonExpiringToken.currentVersion' ./hack/_token.yaml)
+pre_token_version ?= $(shell expr $(token_version) - 1 )
+next_token_version ?= $(shell expr $(token_version) + 1 )
+
+increase-token-version:
+	yq -y '.nonExpiringToken.currentVersion = $(next_token_version)' ./hack/_token.yaml > $(TMPDIR)/_token.yaml
+	mv $(TMPDIR)/_token.yaml ./hack/_token.yaml 
+.PHONY: increase-token-version
+
+refresh-token-version:
+	grep -r -l "ci.openshift.io/token-version: version-$(pre_token_version)" ./clusters | while read file; do $(sed_cmd) -i "s/version-$(pre_token_version)/version-$(token_version)/g" $${file}; done
+.PHONY: refresh-token-version
+
+DRY_RUN ?= server
+
+expire-token-version:
+ifndef EXPIRE_TOKEN_VERSION
+	echo "EXPIRE_TOKEN_VERSION is not defined, existing"
+	false
+endif
+	oc --context app.ci -n ci delete secret -l ci.openshift.io/token-version=version-$(EXPIRE_TOKEN_VERSION)  --dry-run=$(DRY_RUN) --as system:admin
+.PHONY: increase-token-version
+
+list-token-secrets:
+	oc --context app.ci -n ci get secret -l ci.openshift.io/token-version --show-labels
+.PHONY: list-token-secrets
+
+CLUSTER ?= app.ci
+API_SERVER_URL ?= "https://api.ci.l2s4.p1.openshiftapps.com:6443"
+TMPDIR ?= /tmp
+
+config-updater-kubeconfig:
+	$(timeout_cmd) 60 ./clusters/psi/create_kubeconfig.sh "$(TMPDIR)/sa.config-updater.${CLUSTER}.config" ${CLUSTER} $@ ci ${API_SERVER_URL} config-updater-token-version-$(token_version)
+	cat "$(TMPDIR)/sa.config-updater.${CLUSTER}.config"
+.PHONY: config-updater-kubeconfig
+
+secret-config-updater:
```

Comment:
This is something that will have to be called out when adding a new cluster to the build farm. I am not sure that `cluster-init` can handle making the change automatically.